packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -545,8 +597,10 @@ class Cursor extends SynchronousCursor implements ITreeSubscriptionCursor {
  * Returns an implementation of {@link IEditableForest} with no content.
  */
 export function buildForest(
+	breaker: Breakable,
+	schema?: TreeStoredSchemaSubscription,
```

Comment:
We should probably add `@param` comments to this function and, in particular, note what it means for the optional ones to not be provided.

Reply:
This whole function should probably be removed: I've added a TODO about that instead of plumbing the docs through to this API later.